redaxo/src/core/lib/response.php

```diff
@@ -292,13 +292,14 @@ public static function sendContent($content, $contentType = null, $lastModified
             ) {
                 self::sendLastModified($lastModified);
             }
-
             // ----- ETAG
-            if (!self::$sentEtag
-                && (true === rex::getProperty('use_etag') || rex::getProperty('use_etag') === $environment)
-            ) {
-                self::sendEtag($etag ?: self::md5($content));
+            if (self::$sentEtag) {
+                return;
+            }
+            if (!(true === rex::getProperty('use_etag') || rex::getProperty('use_etag') === $environment)) {
```

Comment:
Negierte geklammerte Ausdrücke finde ich sehr schlecht verständlich.

```suggestion
            if (true !== rex::getProperty('use_etag') && rex::getProperty('use_etag') !== $environment)) {
```

Reply:
Das stimmt. Dafür gibts auch ein rector :-)

Das ergebnis hier ist do da ich nur einen einzelnen aktiviert habe